src/renderer/views/History/History.js

```diff
@@ -44,14 +41,15 @@ export default defineComponent({
       dataLimit: 100,
       searchDataLimit: 100,
       doCaseSensitiveSearch: false,
+      ascending: false,
```

Comment:
```suggestion
      useAscendingOrder: false,
```

Recommend adjusting the variable name to be more specific.

Reply:
sounds good, I'll change it